public/app/features/logs/components/getLogRowStyles.ts

```diff
@@ -67,12 +67,16 @@ export const getLogRowStyles = memoizeOne((theme: GrafanaTheme2) => {
       color: ${theme.components.textHighlight.text}
       background-color: ${theme.components.textHighlight};
     `,
```

Comment:
I should ideally migrate this whole module to use object-type style declarations, but I prefer to do it in a separate PR.